src/components/Call/Call.tsx

```diff
@@ -42,12 +42,28 @@ type Call = GetCallQuery["meeting"];
 
 interface Props {
   call: Call;
-  onLeave: (exitType: CallExitType) => void;
+  onLeave: (exitType: CallExitType, droppedStreams: StreamDropType) => void;
   push: (path: string) => void;
   openInfoModal: (resource: FAQResource) => void;
   openTestConnectionModal: () => void;
 }
 
+const streamDropTypes = ["none", "local", "peer", "localAndPeer"] as const;
+export type StreamDropType = typeof streamDropTypes[number];
+const DroppedStreams: Record<StreamDropType, number> = {
+  [streamDropTypes[0]]: 0x00,
+  [streamDropTypes[1]]: 0x01,
+  [streamDropTypes[2]]: 0x02,
+  [streamDropTypes[3]]: 0x03,
+};
+const getDropType = (value: number): StreamDropType => {
+  return (
+    (Object.keys(DroppedStreams) as StreamDropType[]).find(
```

Comment:
nice benefit of the `as const` enum pattern:

```suggestion
    streamDropTypes.find(
```